web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -25,6 +25,14 @@
                         </div>
                     </div>
                     <div class="col-md-12">
+                        <div class="form-group">
+                            <input type="text" class="form-control"
```

Comment:
This could also use a label: maybe something like "Location label", or something along those lines.